user_guide/intro_nav_pinned_pages.rst

```diff
@@ -0,0 +1,61 @@
+.. _user-guide-ui-components-pinned:
+
+Pinbar
+======
+
+Pinning pages is a convenient tool that allows you to save a page and easily return to it. When you open a pinned page, 
+it will look exactly as it did when you have pinned it.
+
+Pins are particularly  useful to:
+
+* Save unfinished forms if you need to switch to another page or simply take a break in your work. All the information 
+  that was in the form at the moment you’ve pinned it will be preserved even if you log out and then log in from another 
+  device.
```

Comment:
Edit: Save unfinished forms if you need to switch to another page or simply take a break in your work. All the information that was in the form at the moment you’ve pinned it will be preserved, even if you log out and then log in from another device.